images/win/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -307,3 +307,9 @@ function Get-SwigVersion {
     $swigVersion = $Matches.Version
     return "Swig $swigVersion"
 }
+
+function Get-ImageMagickVersion {
+    ($(magick -version) | Select-String -Pattern "Version") -match "(?<version>\d+\.\d+\.\d+.\d+)" | Out-Null
```

Comment:
nitpick: remove $() around magick

Reply:
I'll wait for the build and make changes depending on the result. I may need more than this one.
I have a question about this line. Isn't this version specification redundant or insufficient?
An example of outputting the version from the application for understanding: "Version: ImageMagick 7.1.0-48 Q16 x64 c243c92:20220911"